.gitignore

```diff
@@ -43,3 +43,6 @@ jspm_packages
 .env*
 
 .cache
+
+# ide
+.vscode
```

Comment:
i suggest you place this in you system level `~/.gitgnore` so you don't have to do it for every project

Reply:
👍🏿